.github/workflows/build-flannel-images.yml

```diff
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow.json
+name: Build Flannel images
+
+on:
+  workflow_dispatch:
+    inputs:
+      flannel_version:
+        description: 'Version of flannel to build image for (ex: v0.21.5)'
+        required: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build and push images
+      run: |
+        echo "${{ secrets.DOCKER_SECRET }}" | docker login -u ${{ secrets.DOCKER_USERNAME }} --password-stdin
+        cd ./hostprocess/calico && ./build.sh -f ${{ github.event.inputs.flannel_version }}
```

Comment:
Shouldn't the directory be "flannel" instead of "calico"?

Reply:
yes -_-